gear-test/Cargo.toml

```diff
@@ -17,7 +17,6 @@ hex = "0.4.2"
 serde = { version = "*", features = ["derive"] }
 serde_json = "*"
 log = "0.4.16"
-parity-scale-codec = { version = "2.3.1", features = ["derive"], default-features = false }
 sp-keyring = { version = "5.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
```

Comment:
We need to remove all substrate deps from gear-test. They start with `sp-` or `sc-`.

Reply:
Why do we need to remove them at all?

Reply:
Because there is no reason to compile any substrate crates for light-weight in-memory testing. gear-test is tool for fast compile and fast off-chain checking smart contracts execution